### PR TITLE
Add Multiline Text Component

### LINKED ISSUE #95: Implement Multiline Component
Currently, multiline components are not supported, even though we cite them at our README.

### PR DESCRIPTION
**Pull request by Galdineris**
## Motivation
Closes #95

## Modifications
- Added `FWMultilineComponent` folder and respective View, ViewModel and Model.
- Reset cell text in configuration.
- Removed unnecessary comments.
- Fix repeated text from `FWMultilineComponent` and `FWTextComponent`.

## Result
A new component has been added, `FWMultilineComponent`.

##  Checklist
 **This PR is**
* [x] In accordance with our coding principles.
* [x] Implementing tests wherever needed and possible.
* [x] Free of commented code.
* [x] Documented.
* [x] Closing an Issue or related to an Issue
* [x] Approved in the CI

### CODE REVIEW

**Review by csfar — APPROVED** [review]
👏

File: Sources/Formworks/FWComponents/FWMultilineComponent/FWMultilineComponentView.swift [hunk 1]
```diff
@@ -0,0 +1,201 @@
+//
+//  FWMultilineComponentView.swift
+// 
+//
+//  Created by Rafael Galdino on 23/11/20.
+//
+
+import UIKit
+
+final class FWMultilineComponentView: UITableViewCell, FWComponentCell {
```
Comment (EdgSgroi) [inline Sources/Formworks/FWComponents/FWMultilineComponent/FWMultilineComponentView.swift:10]: Should this class be documented?

**Review by EdgSgroi — APPROVED** [review]

**Review by cassiaabarbosa — APPROVED** [review]
Need be commented, mas it is good.

File: Sources/Formworks/FWComponents/FWMultilineComponent/FWMultilineComponentView.swift (hunk 1 above)
Comment (Galdineris) [reply Sources/Formworks/FWComponents/FWMultilineComponent/FWMultilineComponentView.swift:10]: Yes, it should. Unfortunately I couldn't afford to document this before merging due to our deadlines.
I'm creating the issue #98 with all the classes I missed documentation and will work on it as soon as possible.

**Pull request merged**